wrappers/wtsapi32.go

```diff
@@ -0,0 +1,220 @@
+/*
+ * Copyright (c) 2014-2017 MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the license is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package wrappers
+
+import (
+	"syscall"
+	"unsafe"
+)
+
+// Misc consts from WtsApi32.h
+const (
+	CLIENTNAME_LENGTH     = 20
+	DOMAIN_LENGTH         = 17
+	USERNAME_LENGTH       = 20
+	CLIENTADDRESS_LENGTH  = 30
+	WINSTATIONNAME_LENGTH = 32
+)
+
+// WTS_CONNECTSTATE_CLASS enumeration

```

Comment:
The ordering of the constants and structures in this file still need to be adjusted to match the ordering in wtsapi32.h.

Reply:
Now should be ok  ...